pages/common/ctags.md

```diff
@@ -0,0 +1,16 @@
+# ctags
+
+> Generates an index (or tag) file of language objects found in source files for many popular programming languages.
+> More information: <https://ctags.io/>
+
+- Generate tags for a single file, and save them in a file named "tags" in the current directory:
+
+`ctags {{path/to/file}}`
+
+- Generate tags for all files in the current directory and save them in a file with a custom filename:
```

Comment:
This needs to be rewritten.

Reply:
I assume you mean line 10 specifically; what would you suggest? I.e., what's unclear, or what should be explained better?

Reply:
For example like this. I edited my original comment to include a suggestion. It isn't unclear, but I feel it's too long to explain something not so complicated.

Reply:
Perhaps `to a` instead of `in a` there too? I do agree it reads a little better.

Reply:
Agreed, what do you think of phrasing it as "...and output them to a specific file, overwriting the file if it exists"? I thought it might be useful to also mention that the file would be overwritten, as normally a user would expect the program to warn/prompt them in case a file would be overwritten, but ctags doesn't do that.

Reply:
Sounds good to me!